knowledge_repo/app/index.py

```diff
@@ -1,12 +1,17 @@
 from __future__ import absolute_import
 import logging
+
+from nltk.corpus import stopwords
```

Comment:
This won't work if we have not run nltk.download(); which seems messy given our relatively mild dependence on NLTK. Can we just hard code the stopwords?